artifactory/manager.go

```diff
@@ -130,12 +130,11 @@ func (sm *ArtifactoryServicesManager) DeleteProps(params services.PropsParams) (
 	return setPropsService.DeleteProps(params)
 }
 
-func (sm *ArtifactoryServicesManager) UploadFiles(params services.UploadParams) (artifactsFileInfo []utils.FileInfo, totalUploaded, totalFailed int, err error) {
+func (sm *ArtifactoryServicesManager) UploadFiles(params ...services.UploadParams) (artifactsFileInfo []utils.FileInfo, totalUploaded, totalFailed int, err error) {
```

Comment:
The README should be updated with the new API signature.

Reply:
The README is not affected by this change.